### DISCUSSION TITLE
Display Eloquent Collection Model content

### DISCUSSION THREAD

**Discussion by Polfo** [spatie/laravel-ray#85] [Ideas]
`ray()->model($user)` allows to display the attributes of a model. Great.

It would be nice to have something like 

`ray()->models($users)` to display the attributes of a collection of models too.

**Comment by freekmurze** [comment]
`model` can now accept multiple models. You can use that to display a collection of models quickly.

```
ray()->model(...$users)
```

I would accept a PR with tests and docs that add `models($users)` that behaves the same as the code above.
